src/authorization/AuthorizationsDefinition.ts

```diff
@@ -113,7 +113,7 @@ const AUTHORIZATION_DEFINITION: AuthorizationDefinition = {
         resource: Entity.TAG, action: Action.READ,
         attributes: [
           'id', 'userID', 'issuer', 'active', 'description', 'visualID', 'default', 'user.id',
-          'user.name', 'user.firstName', 'user.email'
+          'user.name', 'user.firstName', 'user.email', 'user.issuer'
```

Comment:
Why issuer is added here?

Reply:
I began with a different approach and forgot to remove it. 
Done.